### DISCUSSION TITLE
How to change default color theme ?

### DISCUSSION THREAD

**Discussion by iqlal** [thuliteio/doks#1031] [Q&A]
Hello sorry I'm still a beginner in terms of web dev, I want to ask where can I change the defaut color theme (default purple) to the color I want?

Thanks

**Comment by h-enk** [comment]
If you're using the child theme, first copy `node_modules/@hyas/doks/assets/scss/common/_variables.scss` and paste to `assets/scss/common/_variables.scss`

In `assets/scss/common/_variables.scss`, update [`$primary: $purple;`](https://github.com/h-enk/doks/blob/master/assets/scss/common/_variables.scss#L27)

Note that, the variables for the dark theme are in `assets/scss/common/_dark.scss`

**Reply by iqlal** [reply]
Is it also the same if I use the starter theme?

**Reply by h-enk** [reply]
Yes, that's correct (but forget the first sentence).

**Reply by iqlal** [reply]
Thanks @h-enk
